src/Umbraco.Web.UI.Client/src/views/components/html/umb-control-group.html

```diff
@@ -1,14 +1,16 @@
 <div class="umb-property">
-    <div class="control-group umb-control-group" ng-class="{error: !formValid(), hidelabel:hideLabel=='true'}">
+    <div class="control-group umb-control-group" ng-class="{ error: !formValid(), hidelabel: hideLabel == 'true' }">
         <div class="umb-el-wrap">
-            <label ng-if="hideLabel!=='true'" class="control-label" for="{{alias}}">
-                <span ng-bind-html="labelstring"></span>
-                <span ng-if="required">
-                    <strong class="umb-control-required">*</strong>
-                </span>
-                <small ng-if="descriptionstring">{{descriptionstring}}</small>
-            </label>
-            <div class="controls controls-row" ng-transclude></div>
+            <div ng-if="hideLabel !== 'true'" class="control-header">
```

Comment:
I've added this `control-header` wrapper element so the label and description correctly wrap to their own lines.